.golangci.yml

```diff
@@ -0,0 +1,34 @@
+linters:
+  disable-all: true
+  enable:
+    - bodyclose
+    - deadcode
+    - depguard
+    #- dupl
+    - errcheck
+    #- funlen
+    #- gochecknoinits
+    #- goconst
+    #- gocritic
+    - gocyclo
```

Comment:
Based on my past experiences with cyclomatic complexity and considering some of the rather large functions in our code base (eg `processNextEndpoint` and `reconcileRoutes` in _route.go_), I'm surprised this passes - luckily the default `min-complexity` threshold of 30 is rather large 😄 If you set it to < 20, we start getting failures.  We can enable it although it would be more useful with a lower threshold - I think 20 would be reasonable. `reconcileRoutes` is at 19 so two more  branches would require refactoring but that's not a bad thing 😄 

```
linters-settings:
  gocyclo:
    min-complexity: 20
```

Reply:
Good feedback @tpantelis, thanks! I reduced the threshold to 20 in the latest version.